docs/target-devices-and-platforms.md

```diff
@@ -0,0 +1,45 @@
+# Currently working devices:
+Currently Servonk is targetting Gonk from B2G derived from Android Lollipop. There's also the desktop build with Glutin. 
+
+Using Gonk-L:
+* Sony Xperia Z3 Compact
+* LG Google Nexus 5
+
+Using Glutin:
+* Linux desktop
+
+To do: add links to the build instructions for each of the working devices.
```

Comment:
Let's remove that and open an issue instead.

Reply:
Corrected, pleased excuse me.